### PR TITLE
Unit Tests - Burn Event Service

### LINKED ISSUE #227: Validator Unit Tests
- [x] Helpers
- [x] CMD Clients
- [x] CMD Repositories
- [x] CMD Services
- [x] Model - auth-messages
- [x] Model - burn-event
- [x] Model - messages
- [x] Model - transfer
- [ ] Core - Pair
- [ ] Core - Queue
- [ ] Core - Server
- [ ] Clients - Ethereum
- [ ] Clients - Hedera Node
- [ ] Clients - Mirror Node
- [x] Burn event Service
- [ ] Scheduling Service
- [ ] Contracts service
- [ ] Messages service
- [ ] Signer service
- [ ] Transfers service
- [ ] Persistence 
- [ ] Handlers
- [ ] Watchers
- [ ] Recovery
- [ ] Router

### PR DESCRIPTION
**Pull request by georgiyazovaliiski**
**Detailed description**:
Burn Event Unit Tests with maximum coverage

**Which issue(s) this PR fixes**:
Fixes #227 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: app/helper/hedera/transaction-id.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func ToMirrorNodeTransactionID(txId string) string {
 	return fmt.Sprintf(
 		"%s-%s-%s",
 		accId,
-		split[0],
+		fmt.Sprintf("%09s", split[0]),
```
Comment (Daniel-K-Ivanov) [inline app/helper/hedera/transaction-id.go:36]: why is this change required

**Review by Daniel-K-Ivanov — APPROVED** [review]

**Review by Daniel-K-Ivanov — CHANGES_REQUESTED** [review]
.

File: app/helper/hedera/transaction-id.go (hunk 1 above)
Comment (georgiyazovaliiski) [reply app/helper/hedera/transaction-id.go:36]: Quite often there are timestamps with a lower-digit seconds, therefore an additional 0 appears in the representation, which causes differences which fail data retrieval.

**Review by Daniel-K-Ivanov — APPROVED** [review]
